pkg/planner/cardinality/row_count_index.go

```diff
@@ -432,13 +430,15 @@ func expBackoffEstimation(sctx sessionctx.Context, idx *statistics.Index, coll *
 		}
 		colID := colsIDs[i]
 		var (
-			count      float64
-			err        error
-			foundStats bool
+			count       float64
+			selectivity float64
+			err         error
+			foundStats  bool
 		)
 		if col, ok := coll.Columns[colID]; ok && !col.IsInvalid(sctx, coll.Pseudo) {
 			foundStats = true
 			count, err = GetRowCountByColumnRanges(sctx, coll, colID, tmpRan)
+			selectivity = count / float64(coll.RealtimeCount)
```

Comment:
one case here
if we have a index(normal-col1, json)
when we have normal-col1 > xxx
and we use prefix column stats to est the rows that this mv index can access, that seems also unfair

Reply:
After offline communication, we agree that this logic is reasonable. Because the row count from `GetRowCountByColumnRanges` should correspond to the table total row count instead of the row count of this index.